wgpu/examples/hello/main.rs

```diff
@@ -16,6 +16,9 @@ async fn run() {
             .unwrap()
     };
 
+    #[cfg(target_arch = "wasm32")]
+    log::info!("{:?}", adapter.get_info());
```

Comment:
I feel like we should either always use log - or explain why we're only sometimes using it.

Reply:
Good point! I changed it to use log now in all cases. Or is there a reason log wasn't used in the non-wasm case before?